includes/feature_heatmap_form.inc

```diff
@@ -186,7 +186,9 @@ function generate_heatmap_values() {
       if (strlen($feature_id) > $max_id_len) {
         $max_id_len = strlen($feature_id);
       }
-      $y[] = '<a href="/feature/' . $feature_id . '">' . $feature_id . '</a>';
+      $fid = $expressions[$feature_id]->feature_id;
```

Comment:
Ok, got it. $expressions[$feature_id] is an array (one value for each biomaterial), so we need to take the first element of the array, and then get the ->feature_id. (All elements will have the same feature_id, so might as well take the first)

We can probably just call reset($expressions[$feature_id])->feature_id to get the value.

Reply:
Oh that's a good catch! I updated the code to use reset. Thanks!